### DISCUSSION TITLE
Add a new menu under menu "Help" menu or in menu bar

### DISCUSSION THREAD

**Discussion by hlamranialaoui** [microsoft/vscode-discussions#603] [Extension Development QnA] [answered]
I need to add a new menu "First Kit" in submenu of "help" menu in vscode extension, there is my code in package.json.

"submenus": [
      {
        "id": "firstkitSubmenu",
        "label": "First Kit"
      }
    ],
    "menus": {
      "help": [
        {
          "submenu": "firstkitSubmenu",
          "group": "FirstKitGroup"
        }
      ],
      "firstkitSubmenu": [
        {
          "id": "helpdesk",
          "command": "first-kit.helpdesk",
          "title": "Helpdesk"
        },
        {
          "id": "documentation",
          "command": "first-kit.documentation",
          "title": "Documentation"
        }
      ]
    }
The new menu is not visible!! Currently, I am only able to add my item in the editor context menu in vs code. How can I do that? If not then can we add a submenu item in any of the existing menus? Like could I add a menu item in menu bar?

**Comment by r3m0t** [comment, marked as answer]
It's not possible, check this list- https://code.visualstudio.com/api/references/contribution-points#contributes.menus

**Reply by hlamranialaoui** [reply]
Thank you for the answer, it will be possible one day!!
